src/UmbracoFileSystemProviders.Azure/AzureFileSystem.cs

```diff
@@ -674,28 +662,22 @@ public Stream OpenFile(string path)
         {
             Current.Logger.Debug<AzureBlobFileSystem>($"OpenFile(path) method executed with path:{path}");
 
-            CloudBlockBlob blockBlob = this.GetBlockBlobReference(path);
-
-            if (blockBlob != null)
+            return this.BlobFetch(path, blockBlob =>
             {
                 if (!blockBlob.Exists())
                 {
                     Current.Logger.Info<AzureBlobFileSystem>("No file exists at {Path}", path);
                     return null;
                 }
 
-                MemoryStream stream = new MemoryStream();
-                blockBlob.DownloadToStream(stream);
-
+                Stream stream = blockBlob.OpenRead();
```

Comment:
This change appeared to have been reverted, was that correct, it looked sensible at the time